services/dynamic-sidecar/tests/unit/conftest.py

```diff
@@ -181,3 +184,36 @@ async def cleanup_containers(app: FastAPI) -> AsyncIterator[None]:
         return
 
     await docker_compose_down(app_state.compose_spec, app_state.settings)
+
+
+@pytest.fixture(params=[True, False])
+def volume_has_quota_support(request: FixtureRequest) -> bool:
+    return request.param
+
+
+@pytest.fixture
+def mock_docker_volume(
+    mocker: MockerFixture,
+    monkeypatch: MonkeyPatch,
+    volume_has_quota_support: bool,
+    faker: Faker,
+) -> None:
+    monkeypatch.setenv("AIOCACHE_DISABLE", "1")
+
+    mock_uuid = faker.uuid4()
+    mocker.patch(
+        "simcore_service_dynamic_sidecar.core.docker_utils.uuid4",
+        return_value=mock_uuid,
+    )
+
+    async def _mock_create(*args, **kwargs) -> AsyncMock:
+        if volume_has_quota_support is False:
+            raise aiodocker.DockerError(
+                status=404,
```

Comment:
- Is it really a 404 what is thrown when you cannot create a volume??
- Can you pls use DOCKER_QUOTAS_ERROR_MSG described above in this test?